examples/ruby/spec/drivers/options_spec.rb

```diff
@@ -6,31 +6,44 @@
   describe 'Driver Options' do
     let(:chrome_location) { driver_finder && ENV.fetch('CHROME_BIN', nil) }
 
-  it 'page load strategy normal' do
+    it 'page load strategy normal' do
       options = Selenium::WebDriver::Options.chrome
       options.page_load_strategy = :normal
 
       driver = Selenium::WebDriver.for :chrome, options: options
       driver.get('https://www.google.com')
       driver.quit
-  end
+    end
 
-  it 'page load strategy eager' do
-       options = Selenium::WebDriver::Options.chrome
-       options.page_load_strategy = :eager
+    it 'page load strategy eager' do
+      options = Selenium::WebDriver::Options.chrome
+      options.page_load_strategy = :eager
 
-       driver = Selenium::WebDriver.for :chrome, options: options
-       driver.get('https://www.google.com')
-       driver.quit
-  end
+      driver = Selenium::WebDriver.for :chrome, options: options
+      driver.get('https://www.google.com')
+      driver.quit
+    end
 
-  it 'page load strategy none' do
+    it 'page load strategy none' do
       options = Selenium::WebDriver::Options.chrome
       options.page_load_strategy = :none
 
       driver = Selenium::WebDriver.for :chrome, options: options
       driver.get('https://www.google.com')
       driver.quit
-  end
+    end
+
+    it 'sets remote capabilities' do
+      options = Selenium::WebDriver::Options.firefox
+      options.platform_name = 'Windows 10'
+      options.browser_version = 'latest'
+      cloud_options = {}
+      cloud_options[:build] = my_test_build
+      cloud_options[:name] = my_test_name
+      options.add_option('cloud:options', cloud_options)
+      driver = Selenium::WebDriver.for :remote, url: cloud_url, capabilities: options
+      driver.get('https://www.google.com')
+      driver.quit
+    end
```

Comment:
I think we cannot execute this code. It would error, right? I believe, `cloud_url` is not defined. 

This code can probably stay in the migration guide.

Reply:
> I think we cannot execute this code. It would error, right? I believe, `cloud_url` is not defined.
> 
> This code can probably stay in the migration guide.

You are right, but I added the option to skip it automatically in rspec during execution with an explanation why, in that way we can move the code, and reference it in multiple places without executing the actual test

Let me know if you like this solution, if not I will move the code back where it was